### ISSUE TITLE
Fastchan::sympy 404 not found

### ISSUE THREAD

**Issue by louisdeneve** [fastai/fastconda#35]
I cannot build my docker image because a link used to install the sympy package leads to a 404 page.

More information in this [link](https://community.anaconda.cloud/t/fastchan-sympy-404-not-found/52375).

Hoping you guys will notice this by posting it here.

**Comment by louisdeneve** [comment]
Don't know if this is still up to do date like mentioned [here](https://github.com/fastai/fastconda/issues/33#issuecomment-1104562019), but will '@' you @jph00

**Comment by oguiza** [comment]
Hi @jph00,

I'm having the same issue louisdeneve described above.
I'm trying to create a new conda release for timeseriesAI's `tsai` library using nbdev_conda, but it raises an error when trying to download the sympy package using fastchan (I've created multiple conda releases in the past and it always worked great). 

Here's the issue:
```python
File "/Users/nacho/opt/anaconda3/envs/py37torch113/lib/python3.7/site-packages/conda/core/package_cache_data.py", line 664, in execute
    raise CondaMultiError(exceptions)
conda.CondaMultiError: HTTP 404 NOT FOUND for url <https://conda.anaconda.org/fastchan/osx-64/sympy-1.11.1-py310h2ec42d9_2.tar.bz2>
Elapsed: 00:00.150336
```

The file is shown in anaconda fastchan/packages/sympy, I've checked the file (https://anaconda.org/conda-forge/sympy/1.11.1/download/osx-64/sympy-1.11.1-py310h2ec42d9_2.tar.bz2) but when you click on it you cannot download it you get this message: "The page you are looking for does not exist.".
However, it exists in conda-forge and it can be downloaded (https://anaconda.org/conda-forge/sympy/1.11.1/download/osx-64/sympy-1.11.1-py310h2ec42d9_2.tar.bz2).

**Comment by oguiza** [comment]
I worked around this issue removing the fastchan channel from the channels in the .condarc file (make sure conda-forge is added). sympy is available in conda-forge.

**Comment by sethfri** [comment]
@louisdeneve @oguiza We just fixed a similar issue in #38 by deleting the package, but it looks like a few months after you posted, a `noarch` version of `sympy` was published to `fastchan`. Does that solve your problem or are you still having issues?
